### ISSUE TITLE
Text selection is broken.

### ISSUE THREAD

**Issue by TeffenEllis** [nirrius/kvetch.io#13]
Double clicking or selecting text immediately removes the selection.

**Comment by Jonahss** [comment]
YEah, that's cus as soon as you click on a message, focus gets transferred to the input box.
feature?
Is there a way to detect highlighting text vs just clicking?

**Comment by Jonahss** [comment]
@TeffenEllis ^^

**Comment by TeffenEllis** [comment]
`window.getSelection`
https://developer.mozilla.org/en-US/docs/Web/API/Window.getSelection
I've played with a bit and it's a little temperamental. The method always returns a function with a `type` property indicating if a range has been created.
